Requester/Signer.php

```diff
@@ -113,7 +113,7 @@ public function requestTransaction(TransactionRequest $transactionRequest): Tran
         $flattenedTransactionRequest['documents'] = array_values($flattenedTransactionRequest['documents']);
 
         try {
-            $response = $this->send('requester.requestTransaction', $this->flatten([$flattenedTransactionRequest]));
+            $response = $this->send('requester.requestTransaction', [$flattenedTransactionRequest]);
```

Comment:
Are you sure about this ? I mean :
- that this was indeed the origin of the bug
- that removing this will not create a bug (we do make a change in `flattenedTransactionRequest` that might need more flatten).

If one flatten should be remove it would be the first I think, but this would need to change other things. It needs to be properly debug